tests/Mono.Android-Tests/Directory.Build.targets

```diff
@@ -0,0 +1,43 @@
+<Project>
+
+  <ItemGroup>
+    <TestApkInstrumentation Include="xamarin.android.runtimetests.NUnitInstrumentation">
+      <Package>$(_MonoAndroidTestPackage)</Package>
+      <ResultsPath>$(OutputPath)TestResult-$(_MonoAndroidTestPackage).xml</ResultsPath>
+    </TestApkInstrumentation>
+
+    <TestApkPermission Include="READ_EXTERNAL_STORAGE">
+      <Package>$(_MonoAndroidTestPackage)</Package>
+    </TestApkPermission>
+
+    <TestApkPermission Include="WRITE_EXTERNAL_STORAGE">
+      <Package>$(_MonoAndroidTestPackage)</Package>
+    </TestApkPermission>
+  </ItemGroup>
+
+  <ItemGroup>
+    <AndroidNativeLibrary Include="$(MSBuildThisFileDirectory)libs\arm64-v8a\libreuse-threads.so" />
+    <AndroidNativeLibrary Include="$(MSBuildThisFileDirectory)libs\armeabi-v7a\libreuse-threads.so" />
+    <AndroidNativeLibrary Include="$(MSBuildThisFileDirectory)libs\x86\libreuse-threads.so" />
+    <AndroidNativeLibrary Include="$(MSBuildThisFileDirectory)libs\x86_64\libreuse-threads.so" />
+  </ItemGroup>
```

Comment:
Conceptually, would this item group make more sense to go in `Mono.Android-Test.Shared.projitems`?

Reply:
I considered renaming this to `_NativeLibBuildOutput` and then adding the following to the shared .projitems:
```xml
<AndroidNativeLibrary Include="@(_NativeLibBuildOutput)" />
```
but decided to just move them instead.